docs/tx.md

```diff
@@ -30,18 +30,27 @@ content locally and on Transifex.
   transifex project which is used as a point of comparison for a diff. The
   results of the diff (if any) are put in a new resource in `mup-web` on transifex.
 
-- `pull <resource>`- should be run while on a fresh branch, separate from other
-  code changes. It will grab translated content from the specified resource (branch
-  name). Merges pulled content into `src/trns/po/{localeCode}.po` files. Run for each
-  resource you wish to pull down, usually resources where translation is complete.
+- `pull <resource>`- should be run while on a fresh branch from `master`, separate from other
+  code changes. It will download translated content from of the current project's resources
+  in order of most-recently-updated. Merges pulled content into `src/trns/po/{localeCode}.po` files.
+  Run for each resource you wish to pull down, usually resources where translation is complete.
   Commit result, create PR, and merge PR after Travis build has passed.
 
-- `pushTxMaster` - Updates the master resource - the cannonical copy of translated 
+- `pullAll`- should be run while on a fresh branch from `master`, separate from other
+  code changes. It will download translated content from all resources for the project it
+  is run in in last updated resource order. As each resource is pulled, and then merges
+  pulled content into the associated by locale `src/trns/po/{localeCode}.po` files.
+    * If you'd like to have a history of each change based on resource add a flag of `-c`
+    and it will create a unique commit for each resource that has a change to commit.
+
+  Commit result, create PR, and merge PR after Travis build has passed.
+
+- `pushTxMaster` - Updates the master resource - the canonical copy of translated 
   content. The main usage is for diffing against when examining a new / updated PR.
   This is run whenever there's an update to the master branch. This pushes all
   languages.
 
 - Adding new locales / languages - First make sure that the language has been added to
   the project in Transifex. Create a new `.po` file in `src/trns/po`, copying
   the first three lines (headers) from another `.po` file. Add the locale to
-  `src/util/locales.js` and https://github.com/meetup/meetup-web-platform/blob/master/src/util/localizationUtils.js
\ No newline at end of file
+  `src/util/locales.js` and https://github.com/meetup/meetup-web-platform/blob/master/src/util/localizationUtils.js
```

Comment:
note to self: this utility is being moved in https://github.com/meetup/meetup-web-platform/pull/427